templates/eks/amazon-eks-nodepool-cf.yaml

```diff
@@ -28,6 +28,8 @@ Metadata:
           - NodeVolumeSize
           - KeyName
           - BootstrapArguments
+          - KubeletExtraArguments
```

Comment:
This feels like a BC break to me. What happens in the following scenario?

- Create cluster with the previous version of the template
- Add a whole bunch of custom labels to the node pool
- Update node pool with this template

My assumption is the labels will get lost.

Reply:
We only set the pool name and version labels via Kubelet the rest is handled by Node pool label operator. As we always send labels (pool name & version) in case of a stack update in case of updating with the new template we will set KubeletExtraArguments and doesn't matter the content of BootstrapArguments param, so I think it should be fine. Anyway will test out this scenario.

Reply:
> We only set the pool name and version labels via Kubelet the rest is handled by Node pool label operator.

True. I forgot how that works. Since it's a calculated parameter, I guess it should be fine then.